modules/virt-running-ssp-pipeline-cli.adoc

```diff
@@ -10,87 +10,49 @@ Use a `PipelineRun` resource to run the example pipelines. A `PipelineRun` objec
 
 .Procedure
 
-. To run the Windows 10 installer pipeline, create the following `PipelineRun` manifest:
+. To run the Windows 11 installer pipeline, create the following `PipelineRun` manifest:
 +
 [source,yaml]
 ----
 apiVersion: tekton.dev/v1beta1
 kind: PipelineRun
 metadata:
-  generateName: windows10-installer-run-
+  generateName: windows11-installer-run-
   labels:
-    pipelinerun: windows10-installer-run
+    pipelinerun: windows11-installer-run
 spec:
-  params:
-  - name: winImageDownloadURL
-    value: <link_to_windows_10_iso> <1>
-  pipelineRef:
-    name: windows10-installer
-  taskRunSpecs:
-    - pipelineTaskName: copy-template
-      serviceAccountName: copy-template-task
-    - pipelineTaskName: modify-vm-template
-      serviceAccountName: modify-vm-template-task
-    - pipelineTaskName: create-vm-from-template
-      serviceAccountName: create-vm-from-template-task
-    - pipelineTaskName: wait-for-vmi-status
-      serviceAccountName: wait-for-vmi-status-task
-    - pipelineTaskName: create-base-dv
-      serviceAccountName: modify-data-object-task
-    - pipelineTaskName: cleanup-vm
-      serviceAccountName: cleanup-vm-task
-  status: {}
-----
-<1> Specify the URL for the Windows 10 64-bit ISO file. The product language must be English (United States).
-
-. Apply the `PipelineRun` manifest:
-+
-[source,terminal]
-----
-$ oc apply -f windows10-installer-run.yaml
-----
+    params:
+    -   name: winImageDownloadURL
+        value: ${WIN_IMAGE_DOWNLOAD_URL} <1>
+    -   name: acceptEula
+        value: false <2>
+    pipelineRef:
+        params:
+        -   name: catalog
+            value: redhat-pipelines
+        -   name: type
+            value: artifact
+        -   name: kind
+            value: pipeline
+        -   name: name
+            value: windows-efi-installer
+        -   name: version
+            value: 
+        resolver: hub
+    taskRunSpecs:
+    -   pipelineTaskName: modify-windows-iso-file
+        podTemplate:
```

Comment:
https://tekton.dev/docs/pipelines/pipelineruns/#specifying-taskrunspecs

podtemplate is replaced by taskPodTemplate in v1beta1
@ksimon1  it fails currently with error :
admission webhook "webhook.pipeline.tekton.dev" denied the request: mutation failed: cannot decode incoming new object: json: unknown field "podTemplate"

Reply:
Correct, the object should use v1 instead of v1beta1